components/camel-undertow/src/main/java/org/apache/camel/component/undertow/UndertowEndpoint.java

```diff
@@ -87,6 +92,14 @@
     private boolean optionsEnabled;
     @UriParam(label = "producer")
     private CookieHandler cookieHandler;
+    @UriParam(label = "producer")
```

Comment:
Add websocket as 2nd label, eg "producer,websocket" and do that for the other new websocket options

Reply:
Fixed in a85b3e3